cmd/pn/config.go

```diff
@@ -14,41 +15,42 @@ var (
 )
 
 // Load config from global and user-specific .ini file(s)
-func loadConfig() *conf.ConfigFile {
-	c := conf.NewConfigFile()
-	global, err := os.Open(GlobalConfig)
+func loadConfig(name string) (ini.File, error) {
+	c, err := ini.LoadFile(name)
 	if err == nil {
-		defer global.Close()
-		if err = c.Read(global); err != nil {
-			log.Fatalln("ERROR: reading global config: ", err)
-			return c
-		}
+		return c, nil
 	}
 
-	home := os.Getenv("HOME")
-	user, err := os.Open(filepath.Join(home, UserConfig))
-	if err == nil {
-		defer user.Close()
-		if err = c.Read(user); err != nil {
-			log.Fatalln("ERROR: reading per user config: ", err)
-			return c
-		}
+	// if it doesn't exist, we don't just run with defaults
+	if os.IsNotExist(err) {
+		return make(ini.File), nil
 	}
+	return nil, err
+}
 
-	return c
+func fillMonitoringCommands(config ini.File) {
+	for result := range monitoringResults {
+		if cmd, ok := config.Get("monitoring", result.String()); ok {
+			monitoringCalls[result] = cmd
+		}
```

Comment:
Yes, but only in case of parse errors. Those are handled in loadConfig now. 

Since all of the monitoring event handlers are optional, this looks correct to me.